docs/userguide/containers/dockervolumes.md

```diff
@@ -287,14 +291,16 @@ allows you to upgrade, or effectively migrate data volumes between containers.
 ## Backup, restore, or migrate data volumes
 
 Another useful function we can perform with volumes is use them for
-backups, restores or migrations.  We do this by using the
+backups, restores or migrations.  You do this by using the
 `--volumes-from` flag to create a new container that mounts that volume,
 like so:
 
-    $ docker run --rm --volumes-from dbstore -v $(pwd):/backup ubuntu tar cvf /backup/backup.tar /dbdata
+```bash
+$ docker run --rm --volumes-from dbstore -v $(pwd):/backup ubuntu tar cvf /backup/backup.tar /dbdata
+```
 
 Here we've launched a new container and mounted the volume from the
-`dbstore` container. We've then mounted a local host directory as
+`dbstore` container. You've then mounted a local host directory as
```

Comment:
Changing pronouns here, but still using `we've` in the sentence before, also below.